### ISSUE TITLE
2.15.	Outstanding:MeD: Resources Tabbed interface Focus order (2.4.3) 

### ISSUE THREAD

**Issue by falcochip** [MeD-DMC/Snap-framework-MeD#30]
The focus order for the tabbed interface in the Resources submenu does not work as expected. I was not able to use keyboard navigation to focus and select on Resources.
 
![image](https://user-images.githubusercontent.com/78871138/135318156-d2370e97-ea05-4ede-8a31-0bae2688c152.png)

Consider the focus order in the tabbed interface contained in the [WET working examples](https://wet-boew.github.io/v4.0-ci/demos/tabs/tabs-en.html).
 
![image](https://user-images.githubusercontent.com/78871138/135318184-e5f8a29f-4065-4d65-bd8c-390ad4f7c30d.png)
